src/PhpSpec/ObjectBehavior.php

```diff
@@ -32,6 +32,9 @@
  * @method void beConstructedThrough($factoryMethod, array $constructorArguments)
  * @method void beAnInstanceOf($class)
  * @method void shouldHaveType($type)
+ * @method void shouldImplement($class)
+ * @method void shouldBeAnInstanceOf($class)
+ * @method void shouldReturnAnInstanceOf($class)
```

Comment:
I meant for this specific one I can't see `$this->shouldReturnAnInstanceOf` making sense.

The other three are all good additions